webapp/Connector/src/app/view/module/InteractiveReports.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2016-2017 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+Ext.define('Connector.view.module.InteractiveReports', {
```

Comment:
I was thinking this new component would handle rendering the report content, but seems like what it does is listing the list of reports?

Reply:
This component is listing the public reports stored in core.reports under the heading of "Interactive reports". (This is different that reports listed under the "Reports" section (module/StudyReports.js), which are listing files uploaded on the file server associated with study)
This is the pattern used to display various sections on Study and other pages.